### PR TITLE
Don't dereference None in the click method

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

**Comment by pavpanchekha** [comment]
I'm confused, does this actually happen? As far as I can tell that whole `while` loop is guarded by `elt` being truthy...

**Comment by chrishtr** [comment]
Yes it does happen - see the inner while loop just above the line I'm adding. It repros if you try to load lab13 with examples/example13-width-transition.html and click one of the `<button>` elements.

File: src/lab12.py [hunk 1]
```diff
@@ -283,7 +283,8 @@ def click(self, x, y):
                     if elt.tag == "form" and "action" in elt.attributes:
```
Comment (pavpanchekha) [inline src/lab12.py:283]: Ok, now that I understand the issue, what if we instead change the `while elt` above this to be `while elt.parent`. My point here is that the `if` statement in the outer `while` loop is kind of weird given the condition of that loop. It's really a problem caused by this inner `while` loop so I'd rather fix it there.

**Review by pavpanchekha — APPROVED** [review]
Ok, I suggested what I think is a better fix but this is minor and not worth dragging out, so please pick a fix you prefer and merge.

File: src/lab12.py (hunk 1 above)
Comment (chrishtr) [reply src/lab12.py:283]: Good idea.

**Pull request merged**